brianlong/www.validators.app
# 182958694 api for gossip nodes

#### What's this PR do?
- create api for gossip nodes

#### How should this be manually tested?
- Check the docs
- run script/update_gossip_nodes.rb to get nodes
- check if api endpoint returns correct results

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182958694)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
